### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Oxford-G**
Library Book Project
In this project we build a simple library app with js where you can:

Add books
Remove Books
Mark added books as read

### CODE REVIEW

**Review by umararshadbut — CHANGES_REQUESTED** [review]
# Status: INVALID :red_circle: 

**Hi @Oxford-G **
The readme is descriptive :clap: your commit messages are perfect :clap: However there is missing the pull request description. PLease add the short description in pull request.
I am going to mark this project as invalid so when you fix the issue just submit another code review request.
##

 **If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**Umair**

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+- Javascript
+- HTML
+- CSS
+
+## Getting Started
+
+To set up a local copy of the project
+
+- `git clone git@github.com:Oxford-G/Library.git`
+- `cd Library`
```
Comment (afanimartin) [inline README.md:22]: - How do I run the app after changing directory? Update the `Getting Started` section

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work on this PR. You have fulfilled most of the requirements. Well done!
### Change(s) to make 🙅🏿 🙅🏿 🙅🏿 
- Update the getting started section of the readme. Check comment.


### Martin Afani
#### Slack: whiz25

**Review by umararshadbut — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @Oxford-G ! :+1: 
- Before merge please implement the toggle button funcionality.

**[OPTIONAL]** 
- [ ] Each `Book` displayed has a `button` to `toggle` the `Book` read status ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/toggle_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library))
##
HAPPY CODING :computer:
**Umair**

**Pull request merged**